docs/plugins/discovery-azure.asciidoc

```diff
@@ -1,36 +1,36 @@
-[[discovery-azure]]
+[[discovery-azure-classic]]
 === Azure Discovery Plugin
 
-The Azure Discovery plugin uses the Azure API for unicast discovery.
+The Azure Classic Discovery plugin uses the Azure Classic API for unicast discovery.
```

Comment:
Ok, I wasn't aware of that. :) However, I think a hint on the restriction, a deprecation warning would still be nice. You could maybe link to the PR in the meantime?